sdk/go/azure/compute/managedDisk.go

```diff
@@ -145,6 +145,8 @@ type ManagedDisk struct {
 	OnDemandBurstingEnabled pulumi.BoolPtrOutput `pulumi:"onDemandBurstingEnabled"`
 	// Specify a value when the source of an `Import` or `Copy` operation targets a source that contains an operating system. Valid values are `Linux` or `Windows`.
 	OsType pulumi.StringPtrOutput `pulumi:"osType"`
+	// Whether it is allowed to access the disk via public network. Defaults to `true`.
+	PublicNetworkAccessEnabled pulumi.BoolPtrOutput `pulumi:"publicNetworkAccessEnabled"`
```

Comment:
I'm assuming this is something new from the pulumi side?

Reply:
This is an addition upstream